package.json

```diff
@@ -14,7 +14,7 @@
     "@mui/styled-engine-sc": "^5.8.0",
     "@mui/styles": "^5.11.2",
     "@mui/x-date-pickers": "^5.0.0-alpha.7",
-    "@terraware/web-components": "^2.0.72",
+    "@terraware/web-components": "^2.1.0",
```

Comment:
I think this will be the version number after I merge https://github.com/terraware/web-components/pull/261 but I will update it if I'm wrong.